query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/ids/uuid_create_graphql.rs

```diff
@@ -130,4 +130,62 @@ mod uuid_create_graphql {
 
         Ok(())
     }
+
+    fn schema_cuid_2() -> String {
+        let schema = indoc! {
+            r#"model Todo {
+              #id(id, String, @id, @default(cuid(2)))
+              title String
+            }"#
+        };
+
+        schema.to_owned()
+    }
+
+    // "Creating an item with an id field of model CUIDv2 and retrieving it" should "work"
+    #[connector_test(schema(schema_cuid_2))]
+    async fn create_cuid_v2_and_retrieve_it_should_work(runner: Runner) -> TestResult<()> {
+        let res = run_query_json!(
+            &runner,
+            r#"mutation {
+          createOneTodo(data: { title: "the title" }){
+            id
+          }
+        }"#
```

Comment:
formatting looks a little off to me here

Reply:
Yeah, this was a straight-up copy of the existing test referencing `uuid7`, `create_uuid_v7_and_retrieve_it_should_work`. The whole file is badly formatted tbh

Reply:
The file is now uniformly formatted